src/app/docs/tabs/tabs-docs.component.html

```diff
@@ -4,10 +4,373 @@
   pageTitle="Tabs"
 >
   <sky-docs-demo-page-summary>
-    Sample description.
+    Tabs are containers for dividing large amounts of content on a page.
```

Comment:
I think the tightened up description works well. ... @Blackbaud-SteveBrush, do you think there is any value to putting the old description of `The tabs module renders tabs that divide large subsets of content on a page. It includes options to display buttons in the tab area and to make tabs closeable.` at the top of the Development tab? I don't think so, but just checking before we move on and lose that longer overview.

Reply:
@Blackbaud-SteveBrush and @Blackbaud-AlexKingman, do either of you think it's worth putting this content on the Development tab? If not, we can just resolve this comment.

Reply:
I'm going to assume that the nonresponse here indicates that there's no real interest. I'm resolving the issue.